README.md

```diff
@@ -32,15 +32,16 @@ For using the template with Gazebo Fortress switch to the `fortress` branch of t
     ```
 
 ### Use as template
+
 Directly `Use this template` and create your project repository on Github.
 
-Or start by creating a workspace and cloning the template repository:
+Or start by creating a workspace.
+Use VCS to import `ros_gz_project_template` as well as the dependencies that must be built from source.
 
    ```bash
    mkdir -p ~/template_ws/src
-   cd ~/template_ws/src
-   wget https://raw.githubusercontent.com/gazebosim/ros_gz_project_template/main/template_workspace.yaml
-   vcs import < template_workspace.yaml
+   cd ~/template_ws
```

Comment:
Instead of `~/` do you think `<path_to_workspace>` would be more clear?

Reply:
Yes! I don't think we should assume users call it this, or clone in home directory. I certainly don't.

Reply:
Do you mind making that change, I couldn't make a suggestion over deleted lines